ui/src/queries/useBedSetBedfiles.ts

```diff
@@ -11,19 +11,18 @@ type BedSetBedfilesQuery = {
 
 export const useBedsetBedfiles = (query: BedSetBedfilesQuery) => {
   const { api } = useBedbaseApi();
-  const { id, autoRun } = query;
-  let enabled = false;
-  if (autoRun !== undefined && autoRun && id) {
-    enabled = true;
-  }
+  const { id } = query;
 
   return useQuery({
     queryKey: ['bedset-bedfiles', id],
     queryFn: async () => {
       const { data } = await api.get<BedSetBedfilesResponse>(`/bedset/${id}/bedfiles`);
       return data;
     },
-    enabled: enabled,
-    staleTime: 0,
+    enabled: Boolean(id),
+    staleTime: 5 * 60 * 1000, // Consider data fresh for 5 minutes
+    gcTime: 30 * 60 * 1000, // This replaces cacheTime in newer versions
```

Comment:
Did Chat think this is a good idea, or do we think its a good idea? Do we want to mess with these settings?